packages/react-components/react-teaching-popover-preview/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@fluentui/react-teaching-popover-preview",
+  "version": "0.0.0",
+  "private": true,
+  "description": "New fluentui react package",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "./dist/index.d.ts",
+  "sideEffects": false,
+  "files": [
```

Comment:
doesn't look like you had up to date branch, please update this array 

https://github.com/microsoft/fluentui/blob/master/tools/workspace-plugin/src/generators/react-library/files/package.json__tmpl__#L10-L15